astropy/coordinates/representation/cartesian.py

```diff
@@ -0,0 +1,433 @@
+##############################################################################
+# IMPORTS
+
+from __future__ import annotations
+
+import numpy as np
+from erfa import ufunc as erfa_ufunc
+
+import astropy.units as u
+
+from .base import BaseDifferential, BaseRepresentation
+
+__all__ = [
+    "CartesianRepresentation",
+    "CartesianDifferential",
+]
+
+
+##############################################################################
```

Comment:
No need for this separation, I think

Reply:
Getting rid of them everywhere. It's from my python file template. Nice and organized.

Reply:
✅